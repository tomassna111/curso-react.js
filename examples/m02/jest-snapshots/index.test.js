'use strict'

const respostaQueVemDoServidor = () => [
  'javascript',
  'react',
  'node'
]

it('Test snapshot', () => {
  expect(respostaQueVemDoServidor()).toMatchSnapshot()
})
