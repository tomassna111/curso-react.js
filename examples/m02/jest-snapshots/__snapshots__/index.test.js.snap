// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test snapshot 1`] = `
Array [
  "javascript",
  "react",
  "node",
]
`;
